packages/site/mosaic.config.js

```diff
@@ -38,12 +38,12 @@ module.exports = deepmerge(siteConfig, {
      * Access from your browser as http://localhost:3000/mosaic
      */
     {
-      disabled: process.env.NODE_ENV !== 'development',
+      // disabled: process.env.NODE_ENV !== 'development',
```

Comment:
commented out code

Reply:
removed this code...it was only for testing